compiler/ast/parser.nim

```diff
@@ -1821,6 +1822,10 @@ proc parseSection(p: var Parser, kind: ParsedNodeKind,
   elif p.tok.tokType in {tkSymbol, tkAccent, tkParLe} and p.tok.indent < 0:
     # tkParLe is allowed for ``var (x, y) = ...`` tuple parsing
     result.add defparser(p)
+  elif p.tok.tokType.isKeyword() and p.tok.indent < 0:
+    # in order to make progress accept the unescaped keyword
+    # p.invalidExpectedIdent()
```

Comment:
```suggestion
    # in order to make progress accept the unescaped keyword. The definition
    # parsing is responsible for emitting an error diagnostic, so none is added
    # here
```
For slightly improved clarity.

In addition, I think a branch for `tkKeyWords` also needs to be added to the loop above (line 1808) in order for the following to parse (albeit with errors):
```nim
var
  addr = 0
```

Reply:
Good call.